CONTRIBUTOR_ROLES.md

```diff
@@ -110,12 +109,9 @@ To become a project Maintainer, you must meet the following requirements:
 * You have directly contributed to the project through implementation and/or review.
 * You have been sponsored by two Maintainers.
 
-If you meet the requirements, nominate yourself to become a Maintainer by sending an email to the Maintainers with your candidacy. You must:
+One of the sponsors should open an issue in the relevant repository to provide a nomination. The issue should use the provided nomination template if one exists in that repo. If one does not exist, the sponsor should create a template prior to creating the nomination.
 
-* Ensure that your sponsors are `@mentioned` on the email.
-* Include a list of contributions representative of your work on the project.
-
-Maintainers will vote privately and respond to the issue with either acceptance or with feedback for suggested improvement. Feedback may be given privately.
+Maintainers will vote publically on the issue, expressing their support via a GitHub comment or emoji reaction to the nomination summary. Any concerns may be discussed privately amongst the existing Maintainer team. If feedback needs to be given to the nominee, the sponsor should provide that feedback privately.
```

Comment:
[publically](https://en.wiktionary.org/wiki/publically) is not a wrong word, but `publicly` is more commonly used, and "is generally considered more correct"

Not requesting a change, more of a note as I was wondering why `publically` wasn't in the dictionary. ("This spelling is omitted from many dictionaries")

Reply:
Oh that's my bad! I will change it @cybette thanks for the heads up!